bug_monitor_list_view_inc.php

```diff
@@ -110,6 +110,16 @@
 		<?php } ?>
 	</td>
 </tr>
+<?php if ( ( ON === config_get_global( 'public_urls_enabled' ) ) && isset( $t_share_link ) && ( !is_null( $t_share_link ) ) && ( $t_share_link != '' ) ) { ?>	
```

Comment:
We should not have the URL visible by default or printed when an issue is printed.  But have a page that discloses the URL when a Share button is clicked.  This also reduces changes to core pages.